app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java

```diff
@@ -609,4 +633,12 @@ private void selectImages() {
 
         startActivityForResult(intent, INTENT_REQUEST_GET_IMAGES);
     }
+
+    private boolean isFileExist(String mFilePath) {
```

Comment:
Move this function to fileutils class (https://github.com/Swati4star/Images-to-PDF/blob/master/app/src/main/java/swati4star/createpdf/util/FileUtils.java)